openshift-knative-operator/pkg/serving/kourier.go

```diff
@@ -101,7 +98,8 @@ func addKourierEnvValues(ks base.KComponent) mf.Transformer {
 	}
 
 	networkCM := ks.GetSpec().GetConfig()[networkCMName]
-	if encrypt := networkCM[InternalEncryptionKey]; strings.ToLower(encrypt) == "true" {
+	if strings.ToLower(networkCM[networking.SystemInternalTLSKey]) == "true" ||
```

Comment:
I don't think we need to support the old key longer, we deprecated it in the docs. Also the new value is `enabled` and has a type that can be used. Check out https://github.com/openshift-knative/serverless-operator/pull/2663.